docs/json_schemas/openfga/v1/provider.json

```diff
@@ -0,0 +1,64 @@
+{
+  "title": "ProviderSchema",
+  "description": "Provider schema for OpenFGA.",
+  "type": "object",
+  "properties": {
+    "unit": {
+      "$ref": "#/definitions/BaseModel"
+    },
+    "app": {
+      "$ref": "#/definitions/OpenFGAProviderData"
+    }
+  },
+  "required": [
+    "app"
+  ],
+  "definitions": {
+    "BaseModel": {
+      "title": "BaseModel",
+      "type": "object",
+      "properties": {}
+    },
+    "OpenFGAProviderData": {
+      "title": "OpenFGAProviderData",
+      "type": "object",
+      "properties": {
+        "grpc_api_url": {
+          "title": "GRPC URL",
+          "description": "The url of the grpc API.",
+          "type": "string"
+        },
+        "http_api_url": {
+          "title": "HTTP URL",
+          "description": "The url of the http API.",
+          "type": "string"
+        },
+        "token_secret_id": {
+          "title": "Secret ID of the OpenFGA token",
+          "description": "Secret ID of the preshared token to be used to connect to the OpenFGA service.",
+          "type": "string"
+        },
+        "token": {
+          "title": "Secret ID of the OpenFGA token",
+          "description": "The preshared token to be used to connect to the OpenFGA service, to be used when juju secrets are not available.",
+          "type": "string"
+        },
+        "store_id": {
+          "title": "OpenFGA store ID",
+          "description": "ID of the authentication stored that was created.",
+          "examples": [
+            "01GK13VYZK62Q1T0X55Q2BHYD6"
+          ],
+          "type": "string"
+        }
+      },
+      "required": [
+        "grpc_api_url",
+        "http_api_url",
+        "token_secret_id",
+        "token",
```

Comment:
is token really required? if the token_secret_id is provided, i'd expect this to be empty